### PR TITLE
Add JQuery documentation

### PR DESCRIPTION
**Pull request by anapaulavnoronha**
Quick description about JQuery

### CODE REVIEW

File: _contents/2-libs.md [hunk 1]
```diff
@@ -7,6 +7,19 @@ description: The libraries that we use
 
 ## JQuery
 
+Is an open source JavaScript librabry designed to simplify HTML document traversing,
+event handling, animating, and Ajax interactions. Its slogan is **"Write less, do more"**.
+
+Here some other features that made us choose to use JQuery:
```
Comment (caepenna) [inline _contents/2-libs.md]: We haven't ~chosen~ JQuery, we are just mentioning as one useful tool one might use. You could write "Here are some advantages of using JQuery".

Comment (caepenna) [inline _contents/2-libs.md]: I say we haven't chosen because we use it in very few projects as of now.

File: _contents/2-libs.md [hunk 2]
```diff
@@ -7,6 +7,19 @@ description: The libraries that we use
...
+
+- Is lightweight and easy to use;
+- Handles cross-browser issues;
+- Boosts page load time;
```
Comment (caepenna) [inline _contents/2-libs.md]: In comparison to what?

**Comment by anapaulavnoronha** [comment]
I did the corrections! Thank you, @caepenna

**Comment by caepenna** [comment]
@anapaulavnoronha You can already merge it. Once you've done the corrections just merge it, there's nothing you have to wait on. This has been here for a long time already and that's not good.

**Pull request merged**